app/_components/Date/index.tsx

```
@@ -0,0 +1,15 @@
+import Image from "next/image";
+import styles from "./index.module.css";
+
+type Props = {
+  date: string;
+};
+
+export default function Date({ date }: Props) {
```

Biome (1.9.4)

[error] Do not shadow the global "Date" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)